doc/topics/windows/windows-package-manager.rst

```diff
@@ -334,200 +339,230 @@ you need to install a specific version of a piece of software.
+            ----------
+            7-Zip 9.20 (x64 edition):
+                9.20.00.0
+            Microsoft .NET Framework 4 Client Profile:
+                4.0.30319,4.0.30319
+            Microsoft .NET Framework 4 Extended:
+                4.0.30319,4.0.30319
+            Microsoft Visual C++ 2008 Redistributable - x64 9.0.21022:
+                9.0.21022
+            Mozilla Firefox 17.0.1 (x86 en-US):
+                17.0.1
+            Mozilla Maintenance Service:
+                17.0.1
+            NSClient++ (x64):
+                0.3.8.76
+            Notepad++:
+                6.4.2
+            Salt Minion 0.16.0:
+                0.16.0
+
+    Notice the Full Name for Firefox: ``Mozilla Firefox 17.0.0 (x86 en-US)``.
+    That's exactly what's in the ``full_name`` parameter in the software
+    definition file.
+
+    If any of the software installed on the machine matches one of the software
+    definition files in the repository, the full_name will be automatically
+    renamed to the package name. The example below shows the ``pkg.list_pkgs``
+    for a machine that already has Mozilla Firefox 17.0.1 installed.
+
+    .. code-block:: bash
+
+        test-2008:
+            ----------
+            7zip:
+                9.20.00.0
+            Microsoft .NET Framework 4 Client Profile:
+                4.0.30319,4.0.30319
+            Microsoft .NET Framework 4 Extended:
+                4.0.30319,4.0.30319
+            Microsoft Visual C++ 2008 Redistributable - x64 9.0.21022:
+                9.0.21022
+            Mozilla Maintenance Service:
+                17.0.1
+            Notepad++:
+                6.4.2
+            Salt Minion 0.16.0:
+                0.16.0
+            firefox:
+                17.0.1
+            nsclient:
+                0.3.9.328
+
+    .. important::
+        The version number and ``full_name`` need to match the output from
+        ``pkg.list_pkgs`` so that the status can be verified when running a
+        highstate.
+
+    .. note::
+        It is still possible to successfully install packages using
+        ``pkg.install``, even if the ``full_name`` or the version number don't
+        match. However, this can make troubleshooting issues difficult, so be
+        careful.
+
+:param str installer:
+    The path to the ``.exe`` or ``.msi`` to use to install the package. This can
+    be a path or a URL. If it is a URL or a salt path (``salt://``), the package
+    will be cached locally and then executed. If it is a path to a file on disk
+    or a file share, it will be executed directly.
+
+    .. note::
+        If storing software in the same location as the winrepo it is best
+        practice to place each installer in its own directory rather than the
+        root of winrepo. Then you can place your package definition file in the
+        same directory with a name of ``init.sls``. This will be picked up
+        by ``pkg.refresh_db`` and processed properly.
```

Comment:
@morganwillcock Here's the first part of the warning

Reply:
It isn't just init.sls files which are processed, it is any .sls file.